bin/main.ml

```diff
@@ -869,6 +870,32 @@ let subst =
          )
   , Term.info "subst" ~doc ~man)
 
+let utop =
+  let doc = "Load library in utop" in
+  let man = [ (* TODO *) ] in
+  let go common dir args =
+    let utop_target = dir |> Path.of_string |> Utop.target |> Path.to_string in
+    set_common common ~targets:[utop_target];
+    (* We must wait for other exit hooks to finish before forking. This is
```

Comment:
OK, but then we should make sure no more `at_exit` that needs to be cleaned up is defined. It would be quite annoying to not forget all the finalizers here.